x-pack/plugins/observability_solution/apm/public/components/app/service_groups/service_group_save/group_details.tsx

```diff
@@ -192,12 +209,13 @@ export function GroupDetails({
               iconSide="right"
               onClick={() => {
                 onClickNext({
-                  groupName: name,
+                  groupName: name || '',
```

Comment:
The here you won't need this:
```suggestion
                  groupName: name,
```